hack/prefetch-images.sh

```diff
@@ -0,0 +1,60 @@
+set -ex pipefail
+
+source hack/common.sh
+source cluster-up/cluster/$KUBEVIRT_PROVIDER/provider.sh
+source hack/config.sh
+
+function find_node_names() {
+    if [[ $KUBEVIRT_PROVIDER == "external" ]] || [[ $KUBEVIRT_PROVIDER =~ kind.* ]] || [[ $KUBEVIRT_PROVIDER == "local" ]]; then
+        nodes=() # in case of external provider / kind we have no control over the nodes
+    else
+        nodes=()
+        nodes+=($(_kubectl get nodes -o name | sed "s#node/##g"))
+    fi
+    echo "${nodes[@]}"
+}
+
+function find_pull_command() {
+    if [[ $KUBEVIRT_PROVIDER == "external" ]] || [[ $KUBEVIRT_PROVIDER =~ kind.* ]] || [[ $KUBEVIRT_PROVIDER == "local" ]]; then
+        pull_command="" # in case of external provider / kind we have no control over the nodes
+    else
+        pull_command="docker"
+    fi
+    echo "$pull_command"
+}
+
+function pull_on_nodes() {
```

Comment:
Please prefix the function name with the file name so it will be easy to track what we called.
`function prefetch-images::pull_on_nodes() {`

Same for all other "public" functions.

Reply:
done